services/webserver/50_pushing_to_appuio.rst

```diff
@@ -68,7 +68,7 @@ What happens in this snippet is that we login to APPUiO using the OpenShift CLI,
 
 Important to know is that Gitlab CI will only inject ``KUBE_URL`` and ``KUBE_TOKEN`` as environment variables if the job is classified as a deployment job (which means that it has to contain an ``environment: xyz`` property). For more information about deployment jobs and variables see #2.
 
-The URL to the registry as well as the name of the image we will be building are specified as CI variables in lines 1-3. The custom runner we introduced in the snippet (``image: appuio/gitlab-runner-oc:3.9.0``) simply extends the official ``docker:latest`` with the OC CLI.
+The URL to the registry as well as the name of the image we will be building are specified as CI variables in lines 1-3. The custom runner we introduced in the snippet (``image: appuio/gitlab-runner-oc:3.11.0``) simply extends the official ``docker:latest`` with the OC CLI.
```

Comment:
The image `appuio/gitlab-runner-oc` has not been updated to 3.11 (https://hub.docker.com/r/appuio/gitlab-runner-oc/tags)

Reply:
See appuio/gitlab-runner-oc#6 - after merging create a git tag which will trigger a Docker build

Reply:
@anothertobi @mhutter checked th branch, merged and push a new tag 3.11.0